src/App.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
We should not do this. ESLint is compulsory to keep the code clean and readable. We need to fix all the errors that ESLint throws.